contentcuration/contentcuration/frontend/channelEdit/components/edit/EditView.vue

```diff
@@ -243,6 +243,59 @@
       trackTab(name) {
         this.$analytics.trackClick('channel_editor_modal', name);
       },
+      handleErrorClick(error) {
+        const errorRefs = {
+          Title: 'title',
+          License: 'license',
+          'Copyright holder': 'copyright_holder',
+          Completion: 'randomize',
+          'Learning Activity': 'learning_activities',
+        };
+
+        const errorRef = errorRefs[error];
+
+        const targetElement = this.$refs.detailsTab.$refs[errorRef];
+
+        console.log(targetElement, 'target');
+
+        if (!targetElement) {
+          console.error(`Target element ref not found for error: ${error}`);
+        }
+
+        const nativeElement = targetElement.$el;
+
+        if (nativeElement.scrollIntoView) {
+          nativeElement.scrollIntoView({ behavior: 'smooth' });
+        }
+      },
+      setErrors(errorTags) {
```

Comment:
I see this function receives a nodeId which will be passed to `getNodeDetailsErrorsList`, we can rename this parameter to `nodeId` to make it clearer.